!Command
positional: []
named: []
command:
- wes-server
parent:
subcommands: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/tmp/tmp2wuc40dd/bin/wes-server\"\
  , line 6, in <module>\n    from pkg_resources import load_entry_point\n  File \"\
  /tmp/tmp2wuc40dd/lib/python2.7/site-packages/pkg_resources/__init__.py\", line 3251,\
  \ in <module>\n    @_call_aside\n  File \"/tmp/tmp2wuc40dd/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 3235, in _call_aside\n    f(*args, **kwargs)\n  File \"/tmp/tmp2wuc40dd/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 3264, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n\
  \  File \"/tmp/tmp2wuc40dd/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 585, in _build_master\n    return cls._build_from_requirements(__requires__)\n\
  \  File \"/tmp/tmp2wuc40dd/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 598, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n\
  \  File \"/tmp/tmp2wuc40dd/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 786, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound:\
  \ The 'subprocess32==3.5.2' distribution was not found and is required by wes-service\n"
generated_using:
- --help
